src/lib/models.ts

```diff
@@ -1,13 +1,8 @@
 import * as IR from 'typecheck.macro/dist/IR';
 
-export interface APIGatewayEvent {
```

Comment:
this model is just a simplified version of `APIGatewayProxyEvent` in the aws sdk, so I've replaced it with that